comps/llms/text-generation/vllm-xft/run.sh

```diff
@@ -3,6 +3,9 @@
 # Copyright (C) 2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+# Preload libiomp5.so by following cmd or LD_PRELOAD=libiomp5.so manually
```

Comment:
would be good to add a sentence how libiom5.so is related to fastertransformer. else comment does not match code.